runsc/boot/config.go

```diff
@@ -260,6 +260,12 @@ type Config struct {
 	// parameters to the runtime from docker.
 	TestOnlyTestNameEnv string
 
+	// Coverage files directory path.
```

Comment:
This paragraph could be simplified into something like "CoverageFilesDirectory is the directory where coverage files are generated for the runsc, sandbox, and gofer processes.